daemon/graphdriver/driver_windows.go

```diff
@@ -0,0 +1,18 @@
+// +build windows
+
+package graphdriver
+
+var (
+	// Slice of drivers that should be used in an order
+	priority = []string{
+		"windows",
+	}
+
+	FsNames = map[FsMagic]string{
+		FsMagicUnsupported: "unsupported",
+	}
+)
+
+func GetFSMagic(rootpath string) (FsMagic, error) {
+	return 0, nil
```

Comment:
looks like not applicable to windows or not implemented. (or is it a correct but void implementation?)
in any case, comments here would be nice e.g. `// TODO(windows) not implemented yet`